docs/generated/sparse.COO.abs.rst

```diff
@@ -0,0 +1,6 @@
+COO\.abs
+========
+
+.. currentmodule:: sparse
+
+.. automethod:: COO.abs
```

Comment:
Is there an easy way to auto-generate these files as we change the API?

Reply:
Yes, if we change the API, we just need to delete the `docs/generated` directory and re-run Sphinx with a certain modification. See https://github.com/mrocklin/sparse/pull/43#issuecomment-355033794 for details. However, for now, as RTD will have a standard, unmodified version of Sphinx, we need to include them. Once sphinx-doc/sphinx#4372 is fixed, we don't need to include them in the repository at all, they will be autogenerated.

Reply:
OK.  Perhaps we should have a README or Makefile in the docs directory that explains these steps?  Will this work well with readthedocs?

Reply:
A Makefile is overdo, as we might need to manually modify these files at some point and if the makefile removes them it's no good. We'll probably add howto's for contributions and docs at some point but right now I think it's too early for that.

If these files already exist in the repository, then the _unmodified_ version of Sphinx picks them up and builds the docs nicely. This is actually _why_ I've added them in here. It's just auto-generating these files (specifically `generated/sparse.rst`) that's the issue.